pages/shop/index.js

```diff
@@ -11,59 +11,64 @@ import {
 // import Header from '../../components/header';
 import { Footer } from '../../components/footer';
 import productData from '../../mock/products.json';
+import personData from '../../mock/person.json';
+import NavBar from '../../components/NavBar';
 
 const products = Object.keys(productData);
 const Shop = (props) => {
   const { addCartItem, addShopListItem } = props;
   const { delCartItem, delShopListItem } = props;
   return (
-    <div className="main-container">
-      <div className="content">
-        {/* <Header /> */}
-        <div className="take-to-cart">
-          <button>
-            <Link href="/cart">Take me to cart</Link>
-          </button>
-        </div>
-        <div className="shoppinglist-container">
-          {products.map((itemName) => {
-            return (
-              <ShopCard
-                key={itemName}
-                product={productData[itemName]}
-                quantity={props.shopListItemsCount[itemName] || 0}
-                add={{ addCartItem, addShopListItem }}
-                del={{ delCartItem, delShopListItem }}
-                link={{ href: '/shop/[product]', as: `/shop/${itemName}` }}
-              />
-            );
-          })}
+    <>
+      <NavBar personData={personData} />
+      <div className="main-container">
+        <div className="content">
+          {/* <Header /> */}
+          <div className="take-to-cart">
+            <button>
+              <Link href="/cart">Take me to cart</Link>
+            </button>
+          </div>
+          <div className="shoppinglist-container">
+            {products.map((itemName) => {
+              return (
+                <ShopCard
+                  key={itemName}
```

Comment:
If the `products` array does not contain any unique id, try using this package [uuid](https://www.npmjs.com/package/uuid)